app/main/authorize/keycloak_manager.py

```diff
@@ -31,9 +31,7 @@ def get_user_transferring_body_keycloak_groups(groups):
             continue
         split_str = os.path.split(group)
 
-        if len(split_str) == 1:
-            continue
-
         transferring_body = split_str[1].strip()
-        users_transferring_bodies.append(transferring_body)
+        if len(transferring_body) > 0:
```

Comment:
was there a particular reason for this change?

Reply:
after discussion over the call, no changes required